server/lib/prepaid-budget.js

```diff
@@ -24,6 +24,7 @@ export async function createPrepaidPaymentMethod(originalCreditTransaction) {
     paymentProcessorFeeInHostCurrency: 0,
     platformFeeInHostCurrency: 0,
     hostFeeInHostCurrency: 0,
+    kind: null, // We intentionally set an empty `kind` for prepaid
```

Comment:
@znarf I wonder whether we shouldn't set `ADDED_FUNDS` or create a new `kind` for prepaids? Unless we want to deprecate this approach

Reply:
I thought about it and think we should have a dedicated `kind` for this. `PREPAID_PAYMENT_METHOD` ?

It's something that we want to continue supporting in the future, at least on the mid-term.